config/search/config.json

```diff
@@ -363,6 +363,155 @@
           }
         }
       }
+    },
+    {
+      "type": "newsletter-infos",
+      "on_path": "newsletter-infos",
+      "rdf_type": "http://data.vlaanderen.be/ns/besluitvorming#NieuwsbriefInfo",
+      "properties": {
+        "richtext": "http://mu.semte.ch/vocabularies/ext/htmlInhoud",
+        "subtitle": "http://dbpedia.org/ontology/subtitle",
+        "title": "http://purl.org/dc/terms/title",
+        "publicationDate": [
```

Comment:
I propose to rename `publicationDate` to `meetingDate`. Both can be different things. The path specified here leads to the _meeting_'s date.

Reply:
renamed